setup.cfg

```diff
@@ -1,3 +1,6 @@
+[install]
+optimize=1
```

Comment:
Is that equivalent to python -O? We should avoid it then, pretty much the only thing it does is remove `assert` statements.

Reply:
It's a fix to build .pyc and eventually .pyo files. Otherwise they is an error stating that they are missing while they are listed in INSTALLED_FILES

Reply:
We don't want pyo file as that removes asserts (and that's pretty much all it does).

It should compile to pyc by default but maybe explicitly request it if that doesn't seem to work?

Reply:
I verifyed the rpm content and I don't  see any .pyo files in it

Reply:
OK